go/vt/vtctl/vtctl.go

```diff
@@ -1620,7 +1620,12 @@ func commandCreateKeyspace(ctx context.Context, wr *wrangler.Wrangler, subFlags
 	}
 
 	if !*allowEmptyVSchema {
-		err = wr.TopoServer().EnsureVSchema(ctx, keyspace)
+		cells, err := wr.TopoServer().GetKnownCells(ctx)
```

Comment:
I think we could also just pass an empty cells list to mean "all cells", since `RebuildSrvVSchema` already knows how to expand that by doing this same thing.